Patches.py

```diff
@@ -2097,10 +2102,16 @@ def place_shop_items(rom, world, shop_items, messages, locations, init_shop_id=F
                 if location.item.name == 'Ice Trap':
                     shop_item_name = create_fake_name(shop_item_name)
 
-                if world.world_count > 1:
-                    description_text = '\x08\x05\x41%s  %d Rupees\x01\x05\x42Player %d\x05\x40\x01Special deal! ONE LEFT!\x09\x0A\x02' % (shop_item_name, location.price, location.item.world.id + 1)
+                if (True) or item_display.onetime:
```

Comment:
If shop items could be made repeatable, there would be no (True) there, but they're not so I put it there to prevent it from reaching the else. Maybe shouldn't have left it that way.

Reply:
Yeah, that should be taken out or commented to the effect of why it's this way. Would it make sense to set .onetime to True for all items for the time being?